virtual_workstation_setup.sh

```diff
@@ -355,6 +416,8 @@ pushd "$workdir"
     setup_jira_auth
     setup_gdb
     setup_undodb
+    setup_pipx
```

Comment:
Should we add some kind of comment here explaining the dependency between these two steps?

Reply:
+1. Probably ok to move pipx install closer to the top of this list.

Reply:
I added a comment about the dependency. 

I left pipx install in the same place for now. If we start to use pipx more for installing python package (which seems like a good idea to me), then it might make sense to move it up, but for now I thought it might be nice to keep pipx and the install that requires pipx grouped together.